src/components/footer/template.njk

```diff
@@ -32,8 +32,8 @@
     <div class="govuk-footer__meta">
       <div class="govuk-footer__meta-item govuk-footer__meta-item--grow">
         {% if params.meta %}
+          <h2 class="govuk-visually-hidden">{{ params.meta.title }}</h2>
```

Comment:
At the moment, if we did this it'd be a breaking change since there is no default.

You could consider adding a default using the Nunjucks filter.

`{{ params.meta.title | default("Support links") }}`

You could then keep the existing test the same, and add a new test that sets this new feature.

Reply:
I did debate having fallback text, largely I didn't because if you are switching lang, it might be there by default and not noticed.

Happy to make this change if required though.

Reply:
Yeah I think we want to keep this default since most people will not know they need to set this as it's invisible.